js/components/src/components/dashboard/moderator-panel.tsx

```diff
@@ -0,0 +1,625 @@
+            >
+              {permissions.hide && (
+                <Text style={[textStyle.white, { fontSize: 12 }]}>✓</Text>
+              )}
+            </View>
+            <View>
+              <Text
+                style={[textStyle.white, { fontSize: 14, fontWeight: "500" }]}
+              >
+                Hide Messages
+              </Text>
+              <Text style={[textStyle.gray[400], { fontSize: 12 }]}>
+                Hide individual chat messages
+              </Text>
+            </View>
+          </Pressable>
+
+          <Pressable
+            onPress={() =>
+              setPermissions((p) => ({
+                ...p,
+                "livestream.manage": !p["livestream.manage"],
+              }))
+            }
+            style={[
+              layout.flex.row,
+              layout.flex.alignCenter,
+              p[3],
+              r.md,
+              bg.neutral[800],
+              borders.width.thin,
+              borders.color.neutral[700],
+            ]}
+          >
+            <View
+              style={[
+                {
+                  width: 20,
+                  height: 20,
+                  borderRadius: 4,
+                },
+                borders.width.thin,
+                borders.color.neutral[600],
+                permissions["livestream.manage"]
+                  ? bg.blue[600]
+                  : bg.neutral[900],
+                layout.flex.center,
+                { marginRight: 12 },
+              ]}
+            >
+              {permissions["livestream.manage"] && (
+                <Text style={[textStyle.white, { fontSize: 12 }]}>✓</Text>
+              )}
+            </View>
+            <View>
+              <Text
+                style={[textStyle.white, { fontSize: 14, fontWeight: "500" }]}
+              >
+                Manage Livestream
+              </Text>
+              <Text style={[textStyle.gray[400], { fontSize: 12 }]}>
+                Update stream title
+              </Text>
+            </View>
+          </Pressable>
+        </View>
+      </View>
+
+      {/* Error */}
+      {error && (
+        <View
+          style={[
+            bg.red[900],
+            p[3],
+            r.md,
+            borders.width.thin,
+            borders.color.red[700],
+            mb[4],
+          ]}
+        >
+          <Text style={[textStyle.red[400], { fontSize: 13 }]}>{error}</Text>
+        </View>
+      )}
+
+      {/* Actions */}
+      <DialogFooter>
+        <Button variant="secondary" onPress={onClose} disabled={isLoading}>
+          <Text>Cancel</Text>
+        </Button>
+        <Button
+          variant="primary"
+          onPress={handleAdd}
+          disabled={
+            isLoading ||
+            !moderatorDID.trim() ||
+            Object.values(permissions).every((p) => !p)
+          }
+        >
+          <Text>{isLoading ? "Adding..." : "Add Moderator"}</Text>
+        </Button>
+      </DialogFooter>
```

Comment:
My fault for changing this on y'all but this should be

```suggestion
      <DialogFooter>
        <Button
          width="min"
          variant="secondary"
          onPress={onClose}
          disabled={isLoading}
        >
          Cancel
        </Button>
        <Button
          width="min"
          variant="primary"
          onPress={handleAdd}
          disabled={
            isLoading ||
            !moderatorDID.trim() ||
            Object.values(permissions).every((p) => !p)
          }
        >
          {isLoading ? "Adding..." : "Add Moderator"}
        </Button>
      </DialogFooter>
```

WIll try to find a better solution, I know this is a horrible default.